gravitee-apim-console-webui/src/components/gio-top-nav/gio-top-nav.component.ts

```diff
@@ -92,7 +92,7 @@ export class GioTopNavComponent implements OnInit, OnDestroy {
       .subscribe((settings) => {
         this.portalUrl = isEmpty(settings?.portal?.url)
           ? undefined
-          : this.constants.env.baseURL.replace('{:envId}', this.constants.org.currentEnv.id) + '/portal/redirect';
+          : this.constants.env.baseURL.replace('{:envId}', this.constants.org.currentEnv.id) + '/portal/redirect?version=next';
```

Comment:
Don't merge with this, it's missing a @jourdiw part right ? maybe we can move to Draft ?

Reply:
Correct this should not be in this PR. 

It is still able to be merged as is without any UI changes. I believe that @illichso wants to wait to merge until https://gravitee.atlassian.net/browse/APIM-5348 is ready so that he can test end-to-end.

Reply:
Removed after successful testing